extension/lib/doofinder/Client.js

```diff
@@ -4,19 +4,22 @@ const { promisify } = require('util')
 class Client {
   /**
    * @param {Object} config
-   * @param {string} authHeader
    */
   constructor ({config, tracedRequest, log}) {
     this.baseUri = `https://${config.zone}-search.doofinder.com/5/`
     this.hashId = config.hashId
     this.authKey = config.authKey
+    this.filterMap = config.filterMap
     this.tracedRequest = tracedRequest
     this.log = log
+    this.resulsPerPage = 10
```

Comment:
Why is it hardcoded to 10? 
API supports up to 100: 
```
rpp: "results per page" how many results per page to return. it can't be bigger than 100
```

Reply:
good catch - reworked